src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/DateAdd.txt

```diff
@@ -88,16 +88,16 @@ false
 #Error
 
 >> DateAdd(Date(1, 2, 3), 3)
-Date(1,2,6)
+DateTime(1,2,6,0,0,0,0)
 
 >> Day(DateAdd(Date(2001,1,1), 4, TimeUnit.Days))
 5
 
 >> DateAdd(Date(2011,1,15), 100000000, "milliseconds")
```

Comment:
>100000000 [](http://example.com/codeflow?start=28&length=9)

 Will it overflow if we use 10^12 ms? (it should work as it's ~31 years and a few months)

Reply:
That works fine. Even 10^13 (the result will be around 2327) and 10^14 (5179) would work. Only when we go beyond 9999-12-31 is that we fail